pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -175,6 +189,20 @@ contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
         return _getMiscData().decodeBool(_SWAP_ENABLED_OFFSET);
     }
 
+    /**
+     * @dev Returns true if the allowlist for LPs is enabled.
+     */
+    function getAllowlistLPs() public view returns (bool) {
+        return _getMiscData().decodeBool(_ALLOWLIST_LPS_OFFSET);
+    }
+
+    /**
+     * @dev Verifies that a given address is allowed to hold tokens.
+     */
+    function isAllowedAddress(address member) public view returns (bool) {
+        return false == getAllowlistLPs() || _allowedAddresses[member];
```

Comment:
```suggestion
        return !getAllowlistLPs() || _allowedAddresses[member];
```

Checking equality with false feels weird to me.

Reply:
I've gone back and forth on this. The reason I do this in most languages is clarity. The "!" is easy to miss if you're reading it fast.

Since it doesn't appear elsewhere in the codebase, I can follow the apparent convention and use "!", though spotting negated logic is a lot easier with an explicit "false".

I guess the other way would be to invert the function and say "hasPublicLPs" or something like that, but I think mentioning the allowlist is important.

Reply:
I'll also change the names to make it clearer/consistent with future code in the next PR